### ISSUE TITLE
Rails >= 6?

### ISSUE THREAD

**Issue by futura** [excid3/noticed#44]
The noticed.gemspec says "rails", ">= 6.0.0"

But then, in the readme, you mention "For Rails 6.0 and earlier ... " Does this really just mean "for Rails 6.0"?

I'm just holding out the slim hope that I'll be able to add noticed to my old, gray-bearded Rails 4 project ... unlikely, I know.

**Comment by excid3** [comment]
I don't see that in the readme. What line is that on?

I've tried to make noticed work with any version of Rails that supports ActiveJob, but ran into some snags. We use the ActiveJob serializer to store params in the database and pull them out (like ActiveJob does), but I forget the issue I was running into in 5.2.

We will probably only support 6.0 and up officially. 😕

**Comment by futura** [comment]
Ha! I tried to view "raw" in Github, but it doesn't give me line numbers, so I copied and pasted into my text editor, and it looks like line 397:

`For Rails 6.0 and earlier, you must pass strings of the class names in the `deliver_by` options.`

I do have a new Rails 6.1 app that is communicating with the Rails 4 app, so I still might install it there and build what I need on the other end myself.

I had already built out three separate models for database messages, Twilio notifications and email messages. I love the idea of having these under one domain.
